sdk/file_sdk/OPCFileFormat/OPCFileFormat.pro

```diff
@@ -5,11 +5,20 @@ TARGET    = modopcfile
 TEMPLATE = lib
 CONFIG += staticlib warn_on c++11 debug_and_release
 INCLUDEPATH += $$REPO_ROOT/sdk/rmscrypto_sdk/CryptoAPI $$REPO_ROOT/sdk/file_sdk
+INCLUDEPATH += $$REPO_ROOT/sdk/file_sdk $$REPO_ROOT/sdk/file_sdk/Common
+
+win32:INCLUDEPATH += $$REPO_ROOT/third_party/include/Libgsf
```

Comment:
>32:INCLUDEPATH += $$REPO_ROOT/thi [](start = 3, length = 33)

what about linux?

Reply:
added one for linux, lets see if this will fix issues

---
In reply to: [139361026](https://github.com/AzureAD/rms-sdk-for-cpp/pull/158#discussion_r139361026) [](ancestors = 139361026)

Reply:
it probably won't fix the build because you didn't add the lib.
also, please prepare a Linux vm when you have time because we can't test our changes in the build server until it works

---
In reply to: [139406002](https://github.com/AzureAD/rms-sdk-for-cpp/pull/158#discussion_r139406002) [](ancestors = 139406002,139361026)

Reply:
zip_file.h:4:31: fatal error: common/exceptions.h: No such file or directory

---
In reply to: [139406983](https://github.com/AzureAD/rms-sdk-for-cpp/pull/158#discussion_r139406983) [](ancestors = 139406983,139406002,139361026)

Reply:
next step

---
In reply to: [139424708](https://github.com/AzureAD/rms-sdk-for-cpp/pull/158#discussion_r139424708) [](ancestors = 139424708,139406983,139406002,139361026)